reportResult/index.js

```diff
@@ -1,20 +1,18 @@
 const { getEnv } = require('../lib/env')
 const path = require('path')
 const fs = require('fs')
-const rp = require('request-promise')
+const got = require('got')
 const inquirer = require('inquirer')
 
 async function helloLadok () {
   const pfx = fs.readFileSync(path.resolve(process.cwd(), 'certificate.pfx'))
 
   console.log('Trying to say hello to Ladok...')
 
-  await rp({
+  await got({
```

Comment:
Need some assistance in testing this change - I don't have all the Ladok credentials...
@me97esn @exacs ?